paramiko/buffered_pipe.py

```diff
@@ -101,7 +101,7 @@ def feed(self, data):
             if self._event is not None:
                 self._event.set()
             self._buffer_frombytes(b(data))
-            self._cv.notifyAll()
+            self._cv.notify_all()
```

Comment:
To be sure I understand what's going on here -- it looks like you're updating a number of functions from deprecated CamelCase names to snake-case names, right?

If so -- then, you're sure that all of the snake-case names are available going all the way back to 2.6 & 3.0?

Reply:
Ah, I missed the conversation that's occurring with @jun66j5, please ignore.